DFC.ServiceTaxonomy.CompUi/Handlers/CacheHandler.cs

```diff
@@ -104,7 +104,7 @@ public async Task ProcessRemovedAsync(RemoveContentContext context)
         //Temp check to see if the published items content type matches any of the allowed items in the list
         var pageRouteFlag = PageRouteFlag(processing);
 
-        if (eventGridContentTypes.Any(contentType => context.ContentItem.ContentType.Contains(contentType)) && pageRouteFlag)
+        if (eventGridContentTypes.Any(contentType => context.ContentItem.ContentType.ToLower().Contains(contentType)) && pageRouteFlag)
```

Comment:
May be better to use `.Contains(contentType, StringComparison.OrdinalIgnoreCase)`, then you don't have to think of the casing for any of the items.